src/main/windows/html_email.ts

```diff
@@ -0,0 +1,295 @@
+) {
+  if (open_windows[window_id]) {
+    // window already exists, focus it
+    open_windows[window_id].focus()
+    return
+  }
+  const window = (open_windows[window_id] = new electron.BrowserWindow({
+    backgroundColor: '#282828',
+    // backgroundThrottling: false, // do not throttle animations/timers when page is background
+    darkTheme: true, // Forces dark theme (GTK+3)
+    icon: appIcon(),
+    minHeight: 300,
+    minWidth: 400,
+    show: false,
+    title: `${truncateText(subject, 42)} – ${truncateText(from, 40)}`,
+    height: 600,
+    width: 650,
+    webPreferences: {
+      nodeIntegration: false,
+      preload: join(
+        htmlDistDir(),
+        'electron_html_email_view/electron_html_email_view_preload.js'
+      ),
+      spellcheck: false,
+      webSecurity: true,
+      allowRunningInsecureContent: false,
+      contextIsolation: true,
+    },
+  }))
+  window.webContents.setZoomFactor(DesktopSettings.state.zoomFactor)
+
+  window.webContents.ipc.handle('html_email:get_info', _ => ({
+    subject,
+    from,
+    networkButtonLabelText: tx('load_remote_content'),
+  }))
+
+  nativeTheme.on('updated', () => {
+    window.webContents.ipc.emit('theme-update')
+  })
+
+  window.webContents.ipc.handle(
+    'get-theme',
+    async () => (await loadTheme(DesktopSettings.state.activeTheme)).data
+  )
+
+  window.loadFile(
+    join(
+      htmlDistDir(),
+      'electron_html_email_view/electron_html_email_view.html'
+    )
+  )
+
+  window.webContents.on('will-navigate', (e: electron.Event, _url: string) => {
+    // Prevent drag-and-drop from navigating the Electron window, which can happen
+    // before our drag-and-drop handlers have been initialized.
+    e.preventDefault()
+  })
+
+  window.once('ready-to-show', () => {
+    window.show()
+  })
+
+  window.once('focus', () => {
+    window.webContents.setZoomFactor(DesktopSettings.state.zoomFactor)
+  })
+  window.on('close', () => {
+    context_menu_handle?.()
+    delete open_windows[window_id]
+  })
+
+  let sandboxedView: BrowserView = makeBrowserView(false, htmlEmail)
+  window.setBrowserView(sandboxedView)
+  let context_menu_handle = createContextMenu(window, sandboxedView.webContents)
+
+  window.webContents.ipc.handle(
+    'html-view:resize-content',
+    (_ev, bounds: Electron.Rectangle) => {
+      const window_bounds = window.getBounds()
+      const new_y = Math.floor(window_bounds.height - bounds.height)
+      sandboxedView?.setBounds({
+        ...bounds,
+        y: new_y,
+      })
+    }
+  )
+
+  window.webContents.ipc.handle(
+    'html-view:change-network',
+    async (_ev, allow_network: boolean) => {
+      if (
+        allow_network &&
+        DesktopSettings.state.HTMLEmailAskForRemoteLoadingConfirmation
+      ) {
+        const result = await dialog.showMessageBox(window, {
+          message: tx('load_remote_content_ask'),
+          checkboxLabel: tx('do_not_ask_again'),
+          buttons: [tx('no'), tx('yes')],
```

Comment:
```suggestion
          buttons: [tx('no'), tx('yes')],
          type: "none",
          icon: null,
```

that should remove the icon according to https://www.electronjs.org/docs/latest/api/dialog and https://github.com/electron/electron/issues/13250 - did not try that out, however.

Reply:
thanks for trying in https://github.com/deltachat/deltachat-desktop/pull/3110/commits/65758d2bb7844de158377b324dbae0a95bfd98e3 - then the icon is there, seems to be an electron bug.  was worth a try.